events.go

```diff
@@ -286,26 +286,26 @@ type EventDeploymentFailed struct {
 
 // EventDeploymentInfo describes a 'deployment_info' event.
 type EventDeploymentInfo struct {
-	EventType   string          `json:"eventType"`
-	CurrentStep *DeploymentStep `json:"currentStep"`
-	Timestamp   string          `json:"timestamp"`
-	Plan        *DeploymentPlan `json:"plan"`
+	EventType   string            `json:"eventType"`
+	CurrentStep *DeploymentStepV1 `json:"currentStep"`
```

Comment:
The name `DeploymentStepV1` turned out to be misleading as the structure is already used in pre-1.x. (See also [my comment](https://github.com/gambol99/go-marathon/issues/175#issuecomment-235815659).) We should either rename `DeploymentStepV1` to something more generic that just describes the structure, or rename it to `DeploymentStep` and also rename the _current_ `DeploymentStep`to something different (like `DeploymentStepOld`).

I'll need to ponder about a good choice and good names. Feel free to come up with suggestions too. :-)